charts/filtered-resource-service/templates/pathparam-regex.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2018-2021 Crown Copyright
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+apiVersion: traefik.containo.us/v1alpha1
+kind: Middleware
+metadata:
+  name: filtered-resource-pathparam-regex
+  namespace: {{ include "palisade.namespace" . }}
+spec:
+  replacePathRegex:
+    regex: ^{{ include "filtered-resource-service.root" . }}/filteredResource/(.*)
```

Comment:
using the service name in mixed case as the route url is a terrible idea and kind of exposes function, I recommend making this just /resource

Reply:
See ticket created